exec/cnex/exclude.txt

```diff
@@ -4,7 +4,6 @@ bigint.neon                # assert
 bytes-index.neon           # PUSHPPG
 cal-test.neon              # os.system
 cformat-test.neon          # module
-check-valid.neon           # class
 class-empty.neon           # class
```

Comment:
This test (`class-empty`) ought to work. There are a couple of others that work now that pointers are in place.

Reply:
It does.  I'm going to clean up the exclude.txt file with this build.